content/en/docs/concepts/containers/images.md

```diff
@@ -29,8 +29,7 @@ and possibly a port number as well; for example: `fictional.registry.example:104
 
 If you don't specify a registry hostname, Kubernetes assumes that you mean the Docker public registry.
 
-After the image name part you can add a _tag_ (as also using with commands such
-as `docker` and `podman`).
+After the image name part you can add a _tag_.
```

Comment:
@a-mccarthy I think the original text is OK. After dockershim is removed, people can still use commands like `docker` and `podman` locally with containers, and those analogies will remain relevant.

Reply:
@sftim i was going off the info in the issue, which called out removing references to podman, but i can add them back. I also removed the wording bc i felt like it was more confusing. But if the examples are still valid, i can add them back in